db/db_compaction_test.cc

```diff
@@ -74,6 +74,47 @@ class CompactionStatsCollector : public EventListener {
   std::vector<std::atomic<int>> compaction_completed_;
 };
 
+class DeletionTriggeredCompactionWithMinFileSizeTestListener
+    : public EventListener {
+ public:
+  void OnCompactionBegin(rocksdb::DB* db,
```

Comment:
You may want to remove rocksdb:: to start pleasing https://github.com/facebook/rocksdb/actions/runs/16660590282/job/47156558284?pr=13825 

Local repro command can be found in the log after clicking into the job. It's something like  ASSERT_STATUS_CHECKED=1 TEST_UINT128_COMPAT=1 ROCKSDB_MODIFY_NPHASH=1 LIB_MODE=static OPT="-DROCKSDB_NAMESPACE=alternative_rocksdb_ns" make V=1 -j24 check